app/serializers/curator/filestreams/image_serializer.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
```

Comment:
I'm on the fence about whether we need this class.

Why not just put the `exemplary_image_of` block into `FileSetSerializer`? It doesn't throw an error if used on non-exemplary FileSet subclasses (`Ereader`, for example).

That way, we never have to worry about knowing which serializer needs to be used, since we can use the same one for all FileSet subclasses.

Reply:
The [app/serializers/controlled_terms/nomenclature_serializer.rb](app/serializers/controlled_terms/nomenclature_serializer.rb) is setup the same way. This is to make use in the controllers more manageable. The `FileSetSerializer` is mostly an abstract class that is inheritable. Same with the `NomenclatureSerializer`.